### ISSUE TITLE
Have bim.enable_editing_extrusion_profile() work on all ifcclasses

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#2922]
Would be nice to have `bim.enable_editing_extrusion_profile()` work for all ifcclasses that have an extrusion associated with the shape.

**Comment by Moult** [comment]
Note there is a limitation right now that we only edit the 1st (arbitrarily) extrusion profile. If an object has multiple extrusions, there is no UI yet to edit others / choose which to edit / etc.

Managing multiple extrusions is something that'll take a little more time to cook before we can design a UI for it.
